set.mm

```diff
@@ -26560,6 +26575,21 @@ atomic formula (class version of ~ elsb2 ).  (Contributed by Jim
       BCDKZOPAQLM $.
   $}
 
+  ${
+    $d A x y $.
+    $( The expression ` E. x x = A ` does not depend on a particular choice of
+       the set variable.  This version is proven without ~ df-clab and ~ ax-8 ,
+       but based on ~ ax-9 , ~ ax-ext and ~ df-cleq instead.  Use it in
+       contexts where you want to avoid ~ df-clab , or you need ~ df-cleq
+       anyway.  See ~ cbvexeqsetv-ab , not using ~ df-cleq or ~ ax-ext or
+       ~ ax-9 .  (Contributed by Wolf Lammen, 6-Aug-2025.)
+       (Proof modification is discouraged.) $)
+    cbvexeqsetv $p |- ( E. x x = A <-> E. y y = A ) $=
+      ( cv wceq eqeq1 cbvexvw ) ADZCEBDZCEABHICFG $.
```

Comment:
As I explain in https://us.metamath.org/mpeuni/bj-denotes.html, you can prove that theorem without DV(x,y): simply do it twice with a dummy variable $z$, and use bitri.